3-Authorization-II/3-App-CallAPI/README.md

```diff
@@ -0,0 +1,407 @@
+---
+page_type: sample
+name: Enable your Java Spring Boot daemon to call a protected Web API with its own identity
+description: Enable your Java Spring Boot daemon to call a protected Web API with its own identity
+languages:
+ - java
+products:
+ - azure-active-directory
+ - msal-java
+urlFragment: ms-identity-java-spring-tutorial
+extensions:
+- services: ms-identity
+- platform: Java
+- endpoint: AAD v2.0
+- level: 200
+- client: Java Spring Daemon
+- service: Java Spring Web API
+---
+
+# Enable your Java Spring Boot daemon to call a protected Web API with its own identity
+
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=XXX)
+
+* [Overview](#overview)
+* [Scenario](#scenario)
+* [Prerequisites](#prerequisites)
+* [Setup the sample](#setup-the-sample)
+* [Explore the sample](#explore-the-sample)
+* [Troubleshooting](#troubleshooting)
+* [How to deploy this sample to Azure](#how-to-deploy-this-sample-to-azure)
+* [Next Steps](#next-steps)
+* [Contributing](#contributing)
+* [Learn More](#learn-more)
+
+## Overview
+
+This sample demonstrates a Java Spring Daemon calling a Java Spring Web API that is secured using Azure AD.
```

Comment:
Does it use MSAL Java -if so, we should mention here (and possibly in the title)